opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1390,11 +1390,14 @@ namespace Opm {
                 const double cellVolume = ebosSimulator_.model().dofTotalVolume(cellIdx);
                 auto& cellRes = ebosResid[ cellIdx ];
 
+                const auto& intQuants = *(ebosSimulator_.model().cachedIntensiveQuantities(cellIdx, /*timeIdx=*/0));
+                unsigned pvtRegionIdx = intQuants.pvtRegionIndex();
```

Comment:
probably it's better to use

```c++
unsigned pvtRegionIdx = ebosSimulator_.problem().pvtRegionIndex(cellIdx);
```

here, so that this method does not need to access the intensive quantities...